src/core/Header/Header.tsx

```diff
@@ -16,74 +13,48 @@ import { useTheme } from '../utils/hooks/useTheme';
 import { DropdownMenu } from '../DropdownMenu';
 
 export type HeaderTranslations = {
-  notifications: string;
-  help: string;
-  myAccount: string;
   moreOptions: string;
 };
 
 export type HeaderProps = {
   /**
-   * Icon to the application.
+   * Application title.
+   * (expects `HeaderApplicationButton`)
    */
-  appIcon: React.ReactNode;
+  appButton: React.ReactNode;
   /**
-   * Name of the application.
+   * Content on the right of the application button
+   * (expects `HeaderBreadcrumbs`)
    */
-  appName?: React.ReactNode;
+  breadcrumbs?: React.ReactNode;
   /**
-   * Handler on application click
+   * Content displayed to the left of the user icon
+   * (expects array of `IconButton`, potentially wrapped in a `DropdownMenu`)
    */
-  onApplication?: () => void;
+  actions?: React.ReactNode[];
   /**
-   * `HeaderButton` array, chevron will be added between each.
-   */
-  breadcrumbsButtons?: React.ReactNode[];
-  /**
-   * Content to display in the center section of the header, should handle iui-slim class if used.
-   */
-  centerContent?: React.ReactNode;
-  /**
-   * Expects `UserIcon`, will be given a drop down if `userMenuItems` is also provided;
+   * User icon
+   * (expects a `UserIcon`, potentially wrapped in a `DropdownMenu`)
    */
   userIcon?: React.ReactNode;
-  /**
-   * Handler on notification click, will show notification bell if provided.
-   */
-  onNotification?: () => void;
-  /**
-   * Items in the help dropdown menu.
-   * Pass a function that takes the `close` argument (to close the menu),
-   * and returns a list of `MenuItem` components.
-   */
-  helpMenuItems?: (close: () => void) => JSX.Element[];
-  /**
-   * Items in the user dropdown menu.
-   * Pass a function that takes the `close` argument (to close the menu),
-   * and returns a list of `MenuItem` components.
-   */
-  userMenuItems?: (close: () => void) => JSX.Element[];
   /**
    * Items in the more dropdown menu.
    * Pass a function that takes the `close` argument (to close the menu),
    * and returns a list of `MenuItem` components.
    */
-  moreMenuItems?: (close: () => void) => JSX.Element[];
+  menuItems?: (close: () => void) => JSX.Element[];
   /**
-   * Reduce height of the header;
+   * If true, the header height is reduced, typically used when viewing 3D content.
    * @default false
    */
-  slim?: boolean;
+  isSlim?: boolean;
   /**
    * Provide localized strings.
    */
-  translatedHeader?: HeaderTranslations;
-};
+  translatedStrings?: HeaderTranslations;
+} & React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>;
```

Comment:
I thought iTwinUI-React components allowed using the core component default props, that's why it's there. I can remove them if we dont want to let the users do that. I guess we want to keep `className` ? Are there other props that should be kept ?

Reply:
used `CommonProps`

Reply:
Try Omit<CommonProps, 'title'> because title does not make sense here. This will give you className and style

I don't think we'd need anything else, as long as we are forwarding `...rest` (to allow data- and aria- attributes)